src/templates/navbar-template.js

```diff
@@ -62,18 +62,19 @@ export default function navbarTemplate() {
     ${html`<nav class='nav-scroll' part="navbar-scroll">
       ${(this.hideInfo || !this.resolvedSpec.info)
         ? ''
-        : html`<div class='nav-bar-info' id='link-overview' data-content-id='overview' @click = '${(e) => this.scrollToEventTarget(e, false)}'>
+        : html`<div class='nav-bar-info' id='link-overview' data-content-id='overview' @click = '${(e) => this.scrollToEventTarget(e, false)}' role="link" tabindex="0" @keydown = '${(e) => { if (['Enter', 'Space'].includes(e.key)) { e.target.click(); }}}';
```

Comment:
Question for my own understanding, `Space` seems wrong to me, Space usually controls the scrollbar, is there a reason we adding `Space` to each of these, `Enter` makes sense to me.

Reply:
In the interest of potentially not breaking the usability here in the future, I like the link, that resonates. Can we remove the `Space` for now, and if there is push back on desired usage here we can add it in later without any breaking changes.